composer.json

```diff
@@ -58,9 +59,7 @@
         "fix-cs": "@php ./vendor/bin/php-cs-fixer fix -v --show-progress=estimating"
     },
     "extra": {
-        "_ezplatform_branch_for_behat_tests": "master",
         "branch-alias": {
-            "dev-tmp_ci_branch": "2.3.x-dev",
```

Comment:
We're using the entry for `dev-master` for setting correct branch alias in composer (https://github.com/ibexa/ci-scripts/pull/3/files#diff-caaaa590e98c1cba6c1625f4ff6625f2a5773fd955b477669356ae7655b5f69cR19)